platform/app/services/etl/extractors/workflow_execution_params_service.rb

```
@@ -0,0 +1,81 @@
+        @sync_run = sync_run
+        @sync = sync_run.sync
+        @options = options
+        @workflow_id = generate_workflow_id
+        @options[:workflow_id] = @workflow_id
+        begin
+          @workflow_strategy = workflow_strategy_for(@sync.connection.source.integration_type)
+        rescue Etl::UnsupportedIntegrationType => e
+          log_error(e)
+          raise
+        end
+      end
```

rubocop (1.68.0)

[convention] Method has too many lines. [11/10]

(Metrics/MethodLength)